docs-ui/components/streamGroup.stories.js

```diff
@@ -0,0 +1,207 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {withInfo} from '@storybook/addon-info';
+import {Router, browserHistory, Route} from 'react-router';
+
+import StreamGroup from 'app/components/stream/group';
+import GroupStore from 'app/stores/groupStore';
+
+export default {
+  title: 'Features/Issues/Stream Group',
+};
+
+const selection = {
+  projects: [1],
+  environments: ['production', 'staging'],
+  datetime: {
+    start: '2019-10-09T11:18:59',
+    end: '2019-09-09T11:18:59',
+    period: '',
+    utc: true,
+  },
+};
+const organization = {
+  id: '1',
+  slug: 'test-org',
+  features: [],
+};
+const group = {
+  assignedTo: null,
+  count: '327482',
+  culprit: 'fetchData(app/components/group/suggestedOwners/suggestedOwners)',
+  firstRelease: null,
+  firstSeen: '2020-10-05T19:44:05.963Z',
+  hasSeen: false,
+  id: '1',
+  isBookmarked: false,
+  isPublic: false,
+  isSubscribed: false,
+  lastSeen: '2020-10-11T01:08:59Z',
+  level: 'warning',
+  logger: null,
+  metadata: {function: 'fetchData', type: 'RequestError'},
+  numComments: 0,
+  permalink: 'https://foo.io/organizations/foo/issues/1234/',
+  platform: 'javascript',
+  project: {
+    platform: 'javascript',
+    id: 1,
+    slug: 'test-project',
+  },
+  shareId: null,
+  shortId: 'JAVASCRIPT-6QS',
+  stats: {
+    '24h': [
+      [1517281200, 2],
+      [1517310000, 1],
+    ],
+    '30d': [
+      [1514764800, 1],
+      [1515024000, 122],
+    ],
+  },
+  status: 'unresolved',
+  title: 'RequestError: GET /issues/ 404',
+  type: 'error',
+  userCount: 35097,
+  userReportCount: 0,
+};
+
+class LocationContext extends React.Component {
```

Comment:
unfortunately, this component needed a location in react context AND query params from the router in components that were deeply nested.